app/keepers/keepers.go

```diff
@@ -369,7 +369,6 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 	// if we want to allow any custom callbacks
 	supportedFeatures := "iterator,staking,stargate,osmosis,cosmwasm_1_1"
 
-	wasmOpts = append(owasm.RegisterCustomPlugins(appKeepers.GAMMKeeper, appKeepers.BankKeeper, appKeepers.TwapKeeper, appKeepers.TokenFactoryKeeper), wasmOpts...)
```

Comment:
bank definitely still needs to be there. TokenFactory is moving to general bindings as well, so I think we should keep that for now.

ACK for gamm + twap